ci_microshift.sh

```diff
@@ -5,7 +5,7 @@ set -exuo pipefail
 sudo yum install -y make golang
 
 ./shellcheck.sh
-MICROSHIFT_PRERELEASE=yes ./microshift.sh
+./microshift.sh
```

Comment:
This change shouldn't require because with prerelease flag we get the rpms from mirror.openshift.com